engine/classes/ElggMemcache.php

```diff
@@ -1,111 +1,49 @@
 <?php
+
 /**
  * Memcache wrapper class.
- *
```

Comment:
I decided against deprecating this because we don't have anything to replace it with.